frame/contracts/Cargo.toml

```diff
@@ -40,7 +45,7 @@ sp-std = { version = "4.0.0-dev", default-features = false, path = "../../primit
 [dev-dependencies]
 assert_matches = "1"
 hex-literal = "0.3"
-paste = "1"
+# paste = "1"
```

Comment:
```suggestion
```

Reply:
yep, will remove these once I get a green build that includes the removal of the dev dependencies.

Reply:
removed now we're getting close to the finish line.